docs/api/system-preferences.md

```diff
@@ -35,6 +35,17 @@ example values of `event` are:
 
 Removes the subscriber with `id`.
 
+### `systemPreferences.subscribeLocalNotification(event, callback)` _OS X_
```

Comment:
Do you mind change this to `_macOS_` and rebase the PR? A recent PR just replaced all `OS X` with `macOS`.